### PR TITLE
Changed tests to highlight a bug with contains, fixed the bug in Utils/containsChildrenSubArray

### PR DESCRIPTION
**Pull request by CurtisHumphrey**
moved checker to another line to make linter happy

### CODE REVIEW

File: test/ReactWrapper-spec.js [hunk 1]
```diff
@@ -139,8 +139,8 @@ describeWithDOM('mount', () => {
       ];
 
       const passes = [
-        <span>Hello</span>,
         <div>Goodbye</div>,
+        <span>More</span>,
```
Comment (ljharb) [inline test/ReactWrapper-spec.js]: Could you include both tests (with both orderings, i mean) to ensure we don't regress in the future?

**Comment by CurtisHumphrey** [comment]
Sure I added both cases as passes1 and passes2 @ljharb

**Comment by ljharb** [comment]
Thanks! hmm - i can't find anything in the documentation that talks about whether order should or shouldn't matter if you pass an array of nodes. However, the usage of `some` means that `shallow(<A />).contains([<A />, <B />])` would be expected to pass (which could use a note in the docs) - so I'd expect `shallow(<A />).contains([<B />, <A />])` to pass as well.

@CurtisHumphrey it'd be awesome if you could rebase this on top of latest master - if not, no worries.

@lelandrichardson what do you think about updating the docs to explicitly mention the "any" semantics as well as that ordering doesn't matter?

**Comment by CurtisHumphrey** [comment]
@ljharb I just rebased

**Comment by lelandrichardson** [comment]
@ljharb I think you may have misread this function.  If the render tree contains the entire array being passed in, then it matches. This means that order does matter.  The bug was that this was only working properly if the array in the tree *started* with this sub-array.

Thanks for catching this, @CurtisHumphrey !

**Comment by lelandrichardson** [comment]
@CurtisHumphrey do you mind rebasing and squashing, and I'll merge this in? Thanks!

**Comment by ljharb** [comment]
@lelandrichardson ah, thanks for clarifying. I definitely misread the test, but looking at it again, what you say is indeed represented in the tests :-)

**Comment by CurtisHumphrey** [comment]
@lelandrichardson rebased and squashed

**Pull request merged**